docs/data-ai/capture-data/filter-before-sync.md

```diff
@@ -1,5 +1,5 @@
 ---
-linkTitle: "Filter data"
+linkTitle: "Filter captured data"
```

Comment:
Discuss with Jack probably but I wonder if Selective sync would get better across that the filtering is happening before sync. Maybe Filter before sync is also an option